Competition2018/src/main/java/competition/subsystems/drive/commands/ArcadeDriveWithJoysticksCommand.java

```diff
@@ -30,17 +50,43 @@ public ArcadeDriveWithJoysticksCommand(OperatorInterface oi, DriveSubsystem driv
 
     @Override
     public void initialize() {
-        log.info("Initializing");
+        log.info("Initializing ArcadeDriveWithJoysticksCommand");
         ham.reset();
     }
 
     @Override
     public void execute() {
-        double translate = oi.driverGamepad.getLeftVector().y;
-        double rotate = MathUtils.squareAndRetainSign(oi.driverGamepad.getRightVector().x);
-                
-        double turn = ham.calculateHeadingPower(rotate);
-        driveSubsystem.drive(new XYPair(0, translate), turn);
+        nextTranslateInput = oi.driverGamepad.getLeftVector().y;
+        nextRotateInput = MathUtils.squareAndRetainSign(oi.driverGamepad.getRightVector().x);
+        
+        translateInputDelta = nextTranslateInput - currentTranslateOutput;
+        rotateInputDelta = nextRotateInput - currentRotateOutput;
+        
+        // Rotate
+        if (rotateInputDelta > 0.2 && elevatorSubsystem.getCurrentHeightInInches() > 40) {
```

Comment:
Wrap the whole thing in a check for the elevator height -- we don't need to run any of this if the elevator is below the height limit.